.pre-commit-config.yaml

```diff
@@ -0,0 +1,41 @@
+repos:
+  - repo: https://github.com/PyCQA/flake8
+    rev: 6.0.0
+    hooks:
+      - id: flake8
+        additional_dependencies:
+          - flake8-executable
+          - bandit
+          - flake8-bugbear
+          - flake8-comprehensions
+          - flake8-simplify
+        args: [ "--max-line-length", "120" ]
+        exclude: "tests/to-be-tested/"
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: 'v0.991'
```

Comment:
Could you also enable renovate bot to update the revisions in pre commit? https://docs.renovatebot.com/modules/manager/pre-commit/

Reply:
Sure